beets/dbcore/queryparse.py

```diff
@@ -68,6 +68,10 @@ class is available, `default_class` is used.
     key = match.group(1)
     term = match.group(2).replace('\:', ':')
 
+    if key is not None and term == "":
```

Comment:
`not term` will do fine, as opposed to `term == ""`, since we know it's a string.